dev-tools/mage/target/common/notice.go

```diff
@@ -99,7 +99,7 @@ func Notice() (err error) {
 	}
 	defer f.Close()
 
-	out, err := os.OpenFile(outfn, os.O_WRONLY|os.O_APPEND, 0644)
+	out, err := os.OpenFile(outfn, os.O_RDWR|os.O_APPEND, 0644)
```

Comment:
why this needs to change? what are you trying to solve here @intxgo

Reply:
the file has inconsistent line endings. Now that I am on windows it writes the new lines with windows endings causing CI diff failure, but when I convert the NOTICE.txt file into unix style there are also some changes because the file convent is inconsistent.

Reply:
could you tune your git config to transform CLRF to LF endings on commit?

Reply:
yes, but the problem was that the CI machine didn't produce clean all Unix-style line endings file to compare with the checked-in NOTICE.txt

Reply:
it is green because you added the dos2unix. This wasn't required before, why it is needed now, if you push all the file with LF endings to begin with?